scripts/SchemaToRSTDocumentation.py

```diff
@@ -213,10 +214,14 @@ def buildTableValues(type_map,
 
 
 # Config
-schema_name = 'schema.xsd'
-additional_documentation_name = 'schema.xsd.other'
-complete_output = '../../docs/sphinx/CompleteXMLSchema'
-output_folder = 'docs'
+if len(sys.argv) < 2:
+  print("Usage: %s <schema_dir>" % sys.argv[0])
+  sys.exit(1)
+schema_dir = sys.argv[1]
+schema_name = os.path.join(schema_dir, 'schema.xsd')
+additional_documentation_name = os.path.join(schema_dir, 'schema.xsd.other')
+complete_output = os.path.join(schema_dir, '../../docs/sphinx/CompleteXMLSchema')
+output_folder = os.path.join(schema_dir, 'docs')
```

Comment:
We would benefit from a
```python
def main():
  if len(sys.argv)..
  ...
  ...

if __name__ == "_main_":
  main()
```
Plus some `argparse` and `logging` libs.

Reply:
Yes probably, but I restrained myself from modifying the script beyond what I needed in this PR (just to remove dependency on script's location). There's actually plenty of PEP-8 and similar warnings in it, so if someone pursued a small cleanup in a separate PR that would be nice.

Reply:
@TotoGaz - This script is an old one of mine, so I can work on tidying it up